### ISSUE TITLE
SSL error

### ISSUE THREAD

**Issue by long76** [JetpackDuba/Gitnuro#48]
**Describe the bug**
SSL error by self-signed certificate.

**To Reproduce**
1. clone project by git shell from server with self-signed cert
2. add repo to gitnuro
3. press pull button

**Expected behavior**
pull sucessfull

**Screenshots**
![2022-11-24_09-31-44](https://user-images.githubusercontent.com/18124433/203710598-35704001-34e5-4b0b-a9aa-f2589e1b83ca.png)

**Desktop (please complete the following information):**
 - OS: Kubuntu
 - Version 20.04

maybe possible add trusted store from store - in path `/etc/ssl/certs/ca-certificates.crt`

**Comment by JetpackDuba** [comment]
Thanks for the report!😃

I'm busy for a few weeks but I'll give some feedback ASAP.

**Comment by JetpackDuba** [comment]
Sorry for getting back so late.

Did you use the JAR or the flatpak version?

**Comment by long76** [comment]
> Sorry for getting back so late.
> 
> Did you use the JAR or the flatpak version? 

Jar

**Comment by JetpackDuba** [comment]
Can you try importing the certificate using the keytool utility? As shown here https://stackoverflow.com/a/4326346/4907735

I've used this exact command in the past with Android Studio, I hope it works here too.

**Comment by long76** [comment]
> Can you try importing the certificate using the keytool utility? As shown here https://stackoverflow.com/a/4326346/4907735
> 
> I've used this exact command in the past with Android Studio, I hope it works here too.

yes i know about it but maybe i can just put my certs in folder with jar and program add certs in trust store in runtime? just if i update jdk cacerts will be overrided

**Comment by long76** [comment]
best sostion on this use `ca-certificates-java` package but i can use it in portable jdk

**Comment by JetpackDuba** [comment]
Oke, I'll look again into it in the future as a new feature rather than a bug. I don't know much about certificates and such stuff, but it's probably feasible.

Thanks for your feedback!

**Comment by long76** [comment]
for more usabilty if was ssl error suggest user add repo url in exclude for checking certs. i see similar behavour in https://github.com/eclipsesource/megit

**Comment by JetpackDuba** [comment]
Thanks! I've added the suggestion as part of #30
